db/migrate/20220211155138_add_age_known_to_case_logs.rb

```diff
@@ -0,0 +1,14 @@
+class AddAgeKnownToCaseLogs < ActiveRecord::Migration[7.0]
+  def change
+    change_table :case_logs, bulk: true do |t|
+      t.column :age1_known, :string
+      t.column :age2_known, :string
+      t.column :age3_known, :string
+      t.column :age4_known, :string
+      t.column :age5_known, :string
+      t.column :age6_known, :string
+      t.column :age7_known, :string
+      t.column :age8_known, :string
```

Comment:
All of our other `_known` columns (like `la_known`, `postcode_known`) are integers, I wonder if we would want to keep it the same across these ones as well? 🤔

Reply:
amended